react-assissans/src/components/MovieItem.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Card, Badge } from "react-bootstrap";
+
+function MovieItem(props) {
+  const { item } = props;
```

Comment:
you can pass the {movies} object  immediately MovieItem({movies}) without this line and use it in the return()
ex: 
`<h1>{movies.original_title}</h1>`